pandas/tests/test_strings.py

```diff
@@ -26,6 +27,121 @@ def assert_series_or_index_equal(left, right):
         assert_index_equal(left, right)
 
 
+# method names plus minimal set of arguments to call
+_all_string_methods = ([('get', [0]),
+                        ('join', [',']),
+                        ('contains', ['some_pattern']),
+                        ('match', ['some_pattern']),
+                        ('count', ['some_pattern']),
+                        ('startswith', ['some_pattern']),
+                        ('endswith', ['some_pattern']),
+                        ('findall', ['some_pattern']),
+                        ('find', ['some_pattern']),
+                        ('rfind', ['some_pattern']),
+                        # ('index', ['some_pattern']),
+                        # ('rindex', ['some_pattern']),
```

Comment:
`index` and `partition` and `get_dummies` were having some errors on `Index`. Those will have to be investigated (or xfailed)